ewmarket.py

```diff
@@ -127,9 +127,15 @@ def persist(self):
 				self.donated_poudrins,
 			))
 
+			cursor.execute("DELETE FROM bazaar_wares WHERE {} = %s".format(
+				ewcfg.col_id_server,
+			), (
+				self.id_server,
+			))
+
 			# Write out all current item rows.
 			for name in self.bazaar_wares:
-				cursor.execute("REPLACE INTO bazaar_wares({}, {}, {}) VALUES(%s, %s, %s)".format(
+				cursor.execute("INSERT INTO bazaar_wares({}, {}, {}) VALUES(%s, %s, %s)".format(
```

Comment:
changed how bazaar items are persisted back to how i had it initially so the minigun would be removed when making a new list

Reply:
ok, fair